### PR TITLE
allow to edit tex files

### PR DESCRIPTION
**Pull request by laurentperrinet**
We can already edit a variety of files from within the client - python files or CSS stylesheets.

This simple PR adds the ability to edit TEX files by adding it to the list of ``isEditTextViewSupportedThisFile``

### CODE REVIEW

**Comment by nasli** [comment]
Thanks for your contribution @laurentperrinet!!

**Comment by laurentperrinet** [comment]
Hi! I did as you suggested, I hope it is fine.
Cheers,
Laurent

File: Owncloud iOs Client/Files/PreviewSupportedFiles/Office/OfficeFileView.m [hunk 1]
```diff
@@ -71,7 +71,7 @@ - (void)setFrame:(CGRect)frame {
  * Method to go back in the navigation
  */
 - (void)goBack{
-
```
Comment (nasli) [inline Owncloud iOs Client/Files/PreviewSupportedFiles/Office/OfficeFileView.m]: Please, could you edit the files with the same editor that you did the first time. Now all tabulations in the files are detected as new lines. Thanks @laurentperrinet

**Comment by laurentperrinet** [comment]
I hope I fixed the problem with the extra spaces now
thanks for your patience!

**Comment by nasli** [comment]
Great!! You are Welcome @laurentperrinet ;) 
Just the last thing, please rebase with with master

**Comment by jesmrec** [comment]
Simple checks performed on .tex file:

- [X] tex file icon in file list
- [X] Preview tex file
- [X] Office view tex file
- [X] Edit tex file

Devices: iPhone6Plus v10, iPad Air v9.3.5

Approved

CC @nasli @laurentperrinet

**Pull request merged**